core/templates/dev/head/pages/exploration_editor/editor_tab/ExplorationEditorTabSpec.js

```diff
@@ -163,7 +165,11 @@ describe('State Editor controller', function() {
     it('should initialize the state name and related properties', function() {
       ecs.setActiveStateName('Third State');
       scope.initStateEditor();
-      expect(scs.savedMemento.getHtml()).toEqual('This is some content.');
+      expect(
```

Comment:
This no longer matches the test description. Try and reframe the test accordingly so that the test description and the test contents match.